include/cppjni/jni_wrappers/local_reference.hpp

```diff
@@ -18,39 +17,39 @@ namespace jniwrappers
 {
     template<typename jni_reference_type_t>
     class LocalReference<jni_reference_type_t, boost::hana::when<concepts::is_jni_reference_type<jni_reference_type_t>>>
-        : public boost::noncopyable
     {
     public:
         LocalReference(jni_reference_type_t reference);
         LocalReference(LocalReference&& toMove);
         ~LocalReference();
-        
+
         operator bool() const {return localRef != nullptr;}
-        
+
         LocalReference() = delete;
         void operator=(LocalReference&& toMove) = delete;
-        
+        LocalReference(const LocalReference&) = delete;
+        LocalReference& operator=(const LocalReference& toMove) = delete;
```

Comment:
Variable name again + these constructors should be placed above destructor declaration for clarity.